localization/es/singleton/README.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Singleton
+category: Creational
+language: en
+tag:
+ - Gang of Four
+---
+
+## Intención
+
+Asegurar que una clase solo tenga una instancia y proporcionar un punto de acceso global a ella.
+
+## Explicación
+
+Ejemplo del mundo real
+
+> Solo puede haber una torre de marfil donde los magos estudian su magia. La misma torre de marfil encantada
+> siempre es utilizada por los magos. La torre de marfil aquí es un singleton.
+
+En palabras simples
```

Comment:
Also here, for the context, I think it fits better _En otras palabras_

Reply:
Done